tox.ini

```diff
@@ -53,6 +53,7 @@ envlist =
     kafka-messagebroker_confluentkafka-{py37,py38,py39,py310,py311,py312,py313}-confluentkafkalatest,
     kafka-messagebroker_kafkapython-{py37,py38,py39,py310,py311,py312,py313,pypy310}-kafkapythonlatest,
     kafka-messagebroker_kafkapython-{py38,py39,py310,py311,py312,py313,pypy310}-kafkapythonnglatest,
+    kombu-messagebroker_kombu-{py38,py39,py310,py311,py312,py313,pypy310}-kombulatest,
```

Comment:
It looks like these use rabbitmq, so this line needs to start with that and get sorted next to the other rabbitmq tests.